extension/src/plots/vega/util.ts

```diff
@@ -292,8 +303,16 @@ export const reverseOfLegendSuppressionUpdate = () => ({
   spec: {
     encoding: {
       color: { legend: { disable: false } },
-      shape: { legend: { disable: false } },
-      strokeDash: { legend: { disable: false } }
+      shape: {
+        legend: {
+          disable: false
+        }
+      },
+      strokeDash: {
+        legend: {
+          disable: false
+        }
+      }
```

Comment:
What changed here? Is that only a lint/prettier config?

Reply:
yeah, I can revert